android/java/res/layout/brave_wallet_add_custom_asset.xml

```diff
@@ -0,0 +1,156 @@
+        android:layout_marginEnd="24dp"
+        android:textColor="@color/wallet_text_color"
+        android:text="@string/wallet_add_custom_asset"
+        android:textSize="18sp" />
+
+    <View
+        android:layout_width="match_parent"
+        android:layout_height="1dp"
+        android:layout_marginStart="12dp"
+        android:layout_marginEnd="12dp"
+        android:background="@color/wallet_text_color"
+        android:layout_marginBottom="8dp"/>
+
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="24dp"
+        android:textColor="@color/wallet_text_color"
+        android:text="@string/wallet_add_custom_asset_token_name"
+        android:textSize="14sp" />
+
+    <EditText
+        android:id="@+id/token_name"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="22dp"
+        android:layout_marginEnd="24dp"
+        android:inputType="text"
+        android:textColor="@color/wallet_text_color"
+        android:textSize="14sp"
+        tools:ignore="LabelFor, Autofill" />
+
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="24dp"
+        android:textColor="@color/wallet_text_color"
+        android:text="@string/wallet_add_custom_asset_token_contract_address"
+        android:textSize="14sp" />
+
+    <EditText
+        android:id="@+id/token_contract_address"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="22dp"
+        android:layout_marginEnd="24dp"
+        android:inputType="text"
+        android:textColor="@color/wallet_text_color"
+        android:textSize="14sp"
+        tools:ignore="LabelFor, Autofill" />
+
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="24dp"
+        android:textColor="@color/wallet_text_color"
+        android:text="@string/wallet_add_custom_asset_token_symbol"
+        android:textSize="14sp" />
+
+    <EditText
+        android:id="@+id/token_symbol"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="22dp"
+        android:layout_marginEnd="24dp"
+        android:inputType="text"
+        android:textColor="@color/wallet_text_color"
+        android:textSize="14sp"
+        tools:ignore="LabelFor, Autofill" />
+
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="24dp"
+        android:textColor="@color/wallet_text_color"
+        android:text="@string/wallet_add_custom_asset_decimals"
+        android:textSize="14sp" />
+
+    <EditText
+        android:id="@+id/token_decimals"
```

Comment:
maybe add here default value of decimal
`android:text="18"`
?

Reply:
I made how it looks like on desktop, they don't have any placeholders there

Reply:
thanks